doc/ReleaseChecklist.md

```diff
@@ -4,65 +4,68 @@ Set version variable of the release for further steps, e.g. using the format
- - [ ] update `NOTICE.txt`
- - [ ] update version numbers in [SIRF/CMakeLists.txt](https://github.com/CCPPETMR/SIRF/blob/master/CMakeLists.txt)
- - [ ] update version numbers in the [doc/UsersGuide.md](https://github.com/CCPPETMR/SIRF/blob/master/doc/UserGuide.md) etc
- - [ ] run all demos
- - [ ] run ctest
- - [ ] `git push` 
- - [ ] check Travis
- - [ ] run doxygen, check, send files for uploading and update the doxygen link on Wiki
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
- - [ ] Add release title and description including everything since the last release
- 
-2. SuperBuild
- - [ ] update `CHANGES.md`
- - [ ] update `NOTICE.txt`
- - [ ] update `SIRF-Superbuild/version_config.cmake` with new `SIRF_TAG` (and `STIR_TAG` etc if necessary)
- - [ ] `git push`
- - [ ] check Travis
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
-
-3. Virtual Machine
-
- - [ ] update version number in [VM_version.txt](https://github.com/CCPPETMR/CCPPETMR_VM/blob/master/VM_version.txt)
- - [ ] update `vb.name` in [vagrant/vagrantfile](https://github.com/CCPPETMR/CCPPETMR_VM/blob/master/vagrant/Vagrantfile)
- - [ ] update `CHANGES.md`
- - [ ] update `NOTICE.txt`
- - [ ] `git push`
- - [ ] `vagrant up`
- - [ ] Virtualbox Guest Additions
- - [ ] run `first_run.sh` script (gnome settings and zero-fill trick)
- - [ ] check that the serial port is [deselected](https://github.com/CCPPETMR/CCPPETMR_VM/blob/master/vagrant/README.md#notes-about-ubuntu-box-for-version-100).
- - [ ] export the VM
- - [ ] ctest on VM
- - [ ] upload to website
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
- 
-4. SIRF-Exercises
- - [ ] run all exercises
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
- 
-5. Website
- - [ ] update Software page (version info, VM etc)
- - [ ] upload doxygen
- - [ ] update link for doxygen in [Wiki](https://github.com/CCPPETMR/SIRF/wiki/Software-Documentation)
- - [ ] add news flash
-
-6. Announce
- - [ ] Send email
-
-7. Tag wikis
- - [ ] `git clone https://github.com/CCPPETMR/SIRF.wiki.git; cd SIRF.wiki` (or pull)
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
- - [ ] `git clone https://github.com/CCPPETMR/CCPPETMR_VM.wiki.git; cd CCPPETMR_VM.wiki` (or pull)
- - [ ] `git tag -a v$VER -m "version $VER"`
- - [ ] `git push origin v$VER`
-
+1. Submodules (within SIRF)
+  - [ ] update `data` (dev)
+    + `cd data && git checkout master && git pull`
+    + `git commit -am "update to v$VER" && git tag -a v$VER -m "version $VER"`
+    + `git push && git push --tags`
+    + `cd .. && git add data/ && git commit -m "update data submodule to v$VER"`
+  - [ ] update data (manager)
+    + `git submodule update`
+    + `cd data && git tag -a v$VER -m "version $VER" && cd ..`
+    + `git add data/`  # **should do nothing!**
+    + `git commit -m "update data submodule to v$VER"`  # **should do nothing!**
```

Comment:
not sure about this. I have no idea what `git submodule update` will do when there are changes (hopefully fall-over). If we need a check, I'd therefore rather say "check with `git status`..." (but this would have to be before the tagging of course

Reply:
these lines **should do nothing** since a manager doesn't update code. The dev who last updated the submodule should have run both those lines (albeit with a different commit message).

Reply:
The `submodule update` should also succeed as again the manager shouldn't have altered the submodule. If there are conflicts the submodule won't update but they'd probably get a message.

Reply:
example error:
```bash
SIRF$ git submodule update
error: The following untracked working tree files would be overwritten by checkout:
	...
Please move or remove them before you can switch branches.
Aborting
Unable to checkout 'e4g1...3' in submodule path 'data'
```

Reply:
ok. so why do we have those 2 lines (16-17)  in the checklist? I find them confusing and prone to error (sorry). I'd prefer to have a commit 
```
`git submodule update` #  **should not complain**

Reply:
yes but I presume we should then an explanation as well. Seemed easier to instead include those commands which should make explanation easier. Also I didn't really intend both versions (dev and manager) to survive in this pr - thought you would kill one